api/v1/server/oas/transformers/workflow_run.go

```diff
@@ -549,6 +549,43 @@ func ToScheduledWorkflowsFromSQLC(scheduled *dbsqlc.ListScheduledWorkflowsRow) *
 	return res
 }
 
+func ToCronWorkflowDetailsFromSQLC(cron *dbsqlc.GetCronWorkflowByIdRow) *gen.CronWorkflows {
+	var additionalMetadata map[string]interface{}
+
+	if cron.AdditionalMetadata != nil {
+		err := json.Unmarshal(cron.AdditionalMetadata, &additionalMetadata)
+		if err != nil {
+			return nil
+		}
+	}
+
+	var input map[string]interface{}
+
+	if cron.Input != nil {
+		err := json.Unmarshal(cron.Input, &input)
+		if err != nil {
+			return nil
+		}
+	}
+
+	res := &gen.CronWorkflows{
+		Metadata:           *toAPIMetadata(sqlchelpers.UUIDToStr(cron.ID_2), cron.CreatedAt_2.Time, cron.UpdatedAt_2.Time),
```

Comment:
nit: I think we should modify the query to explicitly select which created at we want here, since this seems very bug prone to me

Reply:
i think i did already